kotlin-reference-server/src/main/kotlin/org/stellar/reference/sep24/Sep24Helper.kt

```diff
@@ -99,6 +99,7 @@ class Sep24Helper(private val cfg: Config) {
     transaction.sign(cfg.sep24.keyPair)
 
     val resp = server.submitTransaction(transaction)
+    log.info("transaction submitted: ${resp.decodedTransactionResult}")
```

Comment:
```suggestion
    info("transaction submitted: ${resp.decodedTransactionResult}")
```

Reply:
Ah, this was only for debugging. The string that it ends up printing is not that useful.